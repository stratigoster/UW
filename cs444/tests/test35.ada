PACKAGE foo IS
  function "+" (a,b: IN integer) return boolean;
  function "+" (a,b: OUT integer) return integer; 
--                   ^^^
-- functions only take IN parameters

  function "*" (a,b: IN OUT integer) return integer; 
--                   ^^^^^^
-- functions only take IN parameters
END foo;
