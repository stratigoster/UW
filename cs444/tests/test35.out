***ERROR***:3:34:	Function arguments can't have mode OUT
***ERROR***:7:37:	Function arguments can't have mode OUT

------
Compilation error
------
